examples/web-components-kitchen-sink/src/stories/misc/script/script.stories.ts

```diff
@@ -1,17 +1,19 @@
+/* eslint-disable storybook/hierarchy-separator */
+// @TODO: discuss about having dots in a meta title (that is not related to a separator)
```

Comment:
There are plenty of reasons why people might want to do this (e.g. "java-style" component names). I think people might just disable the rule entirely if they want to do something like that though.

Reply:
Makes sense. Fixed by ignoring the dot in the eslint rule!